Product/Production/Adapters/General/CONNECTAdapterWeb/pom.xml

```diff
@@ -19,16 +19,6 @@
         </dependency>
         <dependency>
             <groupId>org.connectopensource</groupId>
-            <artifactId>SharedEngineCore</artifactId>
-            <version>${project.parent.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>org.connectopensource</groupId>
-            <artifactId>AdapterAuthenticationCore</artifactId>
-            <version>${project.parent.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>org.connectopensource</groupId>
             <artifactId>PatientDiscoveryCore</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
```

Comment:
We use PolicyEngineCore still in AdapterWeb.  We don't use redaction engine, so I removed that.